src/backend/tasks_io/datafeeds/datafeed_fms_api.py

```diff
@@ -413,14 +387,12 @@ def _get_event_short(self, event_short: str, event: Optional[Event]) -> str:
         if event and event.first_code:
             return event.first_code
 
-        # Second, check if it is a new 2023 division event code
-        if event and event.year == 2023:
-            return DatafeedFMSAPI.EVENT_SHORT_EXCEPTIONS_2023.get(
-                event_short, event_short
-            )
+        # Second, use the first_api_code if the event exists
+        if event and event.first_api_code:
+            return event.first_api_code
 
         # Otherwise, check hard-coded exceptions
-        return DatafeedFMSAPI.EVENT_SHORT_EXCEPTIONS.get(event_short, event_short)
+        return Event.EVENT_SHORT_EXCEPTIONS.get(event_short, event_short)
```

Comment:
Oh i meant we dont need this either.

I think all usages of `_get_event_short` can just be `event.first_api_code`?